common/scaffolding/typographie.html

```diff
@@ -0,0 +1,39 @@
+---
+{
+	"title": "Typographie",
+	"language": "fr",
+	"altLangPage": "typography.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-08-24"
+}
+---
+
+<h1>Titres</h1>
+<h1>Il s'agit d'un élément de titre h1</h1>
+<h2>Il s'agit d'un élément de titre h2</h2>
+<h3>Il s'agit d'un élément de titre h3</h3>
+<h4>Il s'agit d'un élément de titre h4</h4>
+<h5>Il s'agit d'un élément de titre h5</h5>
+<h6>Il s'agit d'un élément de titre h6</h6>
+<pre><code>
+&lt;h1&gt;Il s'agit d'un élément de titre h1&lt;/h1&gt;
+&lt;h2&gt;Il s'agit d'un élément de titre h2&lt;/h2&gt;
+&lt;h3&gt;Il s'agit d'un élément de titre h3&lt;/h3&gt;
+&lt;h4&gt;Il s'agit d'un élément de titre h4&lt;/h4&gt;
+&lt;h5&gt;Il s'agit d'un élément de titre h5&lt;/h5&gt;
+&lt;h6&gt;Il s'agit d'un élément de titre h6&lt;/h6&gt;
+</code></pre>
+
+<h1>Paragraphe</h1>
```

Comment:
Except for a few exception, a page should only contain one heading level 1
```suggestion
<h2>Paragraphe</h2>
```

Reply:
Fixed and addressed in commit "Updated headings and added paragraph".